deploy/examples/cluster.yaml

```diff
@@ -70,6 +70,8 @@ spec:
     # port: 8443
     # serve the dashboard using SSL
     ssl: true
+    # prometheusEndpoint: <protocol>://<prometheus-host>:<port>
+    # prometheusEndpointSSLVerify: false
```

Comment:
Since they include the protocol in the URL, can we automatically determine if ssl is enabled because of `https` or `http` and no need for the ssl verify setting? Or does that setting represent some other behavior?
```suggestion
```

Reply:
`PROMETHEUS_API_SSL_VERIFY` is set to `False, if someone is using a self-signed certificate. If the user has a CA approved certificate, then it can be `True` and the ssl verification will be a success. So its a different one and its useful for people who are mostly using the self-signed CAs

Reply:
@travisn the `prometheusEndpointSSLVerify` flag is not detect whether are we using `http` or `https`, it's to enable/disable server SSL certificate check when `https` is used. Some customers may have valid certificates on their Prometheus server so in this case they should enable this flag, others (using self-signed certificates i.e) will need to keep disabled because otherwise dashboard will complain when checking the SSL certificate.